### DISCUSSION TITLE
Not updating to .19

### DISCUSSION THREAD

**Discussion by Docfunkinstein** [mbround18/valheim-docker#445] [Help!] [answered]
server has auto update turned on, and its sitting at .14 release.  shut off the container and removed the image and redeployed and still only on .14

Is there something else I need to do?

**Comment by mbround18** [comment, marked as answer]
@Docfunkinstein Unfortunately, you will just have to wait until they push the server version. They often push the update to the game on steam before they push the update to the dedicated servers... Sometimes it can take upwards of a few hours before the updates able to be downloaded.

**Reply by Docfunkinstein** [reply]
Good to know, as I really couldn't see what I was doing wrong.

**Reply by mbround18** [reply]
Looks like they took a while to update the server

Patch notes went out 7am yesterday
![Screenshot_20210918-180209.jpg](https://user-images.githubusercontent.com/12646562/133912257-cf93b4f9-6e6c-4c71-b1ff-177167a0d193.jpg)


Server didn't get the update until 1-2am 
![Screenshot_20210918-180219.jpg](https://user-images.githubusercontent.com/12646562/133912265-c45cf032-1db6-411a-9e5d-814693260038.jpg)
